python/ray/train/huggingface/utils.py

```diff
@@ -19,6 +21,43 @@
     from ray.data.preprocessor import Preprocessor
 
 
+@PublicAPI(stability="alpha")
+def to_air_checkpoint(
+    model: Union[transformers.modeling_utils.PreTrainedModel, torch.nn.Module],
+    tokenizer: Optional[transformers.PreTrainedTokenizer] = None,
+    preprocessor: Optional["Preprocessor"] = None,
+    path: Optional[str] = None,
+) -> Checkpoint:
+    """Convert a pretrained Transformers model to AIR checkpoint for serve or inference.
+
+    Args:
+        model: Either a ``transformers.PreTrainedModel``, or a trained PyTorch model.
+        tokenizer: Tokenizer to be used in the Transformers pipeline
+            during serving/inference.
+        preprocessor: A fitted preprocessor. The preprocessing logic will
+            be applied to the inputs for serving/inference.
+        path: The directory where the checkpoint will be stored to.
```

Comment:
whose responsibility it is to delete this path in the future if it's temporary?

Reply:
is it usually the case that `model` is coming from a file already and we are writing it back to another file?

Reply:
I would imagine if the user doesn't supply anything, they wouldn't want to manage it either. Then it's left to nobody to clean it up and may blow up the disk.

Reply:
I'll use explicit path for now, let's discuss later!